doc/guides/west/without-west.rst

```diff
@@ -3,30 +3,58 @@
-This section applies to Zephyr users who fall into one or more of these
-categories:
+It is possible to do Zephyr development without installing west.
+However, you will have to do a lot more work "by hand" to replace its
+features. This page documents the details.
 
-- Already use a multi-repository management system (for example Google repo)
+Getting the Source
+------------------
 
-- Do not need additional functionality beyond compiling and linking
-
-In order to obtain the Zephyr source code and build it without the help of
-west you will need to manually clone the repositories listed in the
-`manifest file`_ one by one, at the path indicated in it.
+In addition to downloading the zephyr source code repository itself,
+you will need to manually clone the additional projects listed in the
+:term:`west manifest` file inside that repository.
 
 .. code-block:: console
 
    mkdir zephyrproject
    cd zephyrproject
    git clone https://github.com/zephyrproject-rtos/zephyr
-   # clone additional repositories listed in the manifest file,
-   # and check out the specified revisions
+   # clone additional repositories listed in zephyr/west.yml,
+   # and check out the specified revisions as well.
+
+As you pull changes in the zephyr repository, you will also need to
+maintain those additional repositories, adding new ones as necessary
+and keeping existing ones up to date at the latest revisions.
+
+Specifying Modules
+------------------
+
+If you have west installed, the Zephyr build system will use it to set
+:ref:`ZEPHYR_MODULES <important-build-vars>`. If you don't have west
+installed and your application does not need any of these
+repositories, the build will still work.
+
+If you don't have west installed and your application *does* need one
+of these repositories, you must set :makevar:`ZEPHYR_MODULES`
+yourself. You can do this at CMake time, e.g. with a command line like
+``cmake -DZEPHYR_MODULES=path/to/repository1;path/to/repository2
+[...]``. You will need to maintain changes to this list as additional
+code is migrated out of zephyr's :file:`ext` directory into separate
+Git repositories.
 
-If you want to manage Zephyr's repositories without west but still need to
-use west's additional functionality (flashing, debugging, etc.) it is possible
-to do so by manually creating an installation:
+Flashing and Debugging
+----------------------
+
+Running build system targets like ``ninja flash``, ``ninja debug``,
+etc. is just a call to the corresponding :ref:`west command
+<west-build-flash-debug>`. If you don't have west installed on your
+system, those targets will error out. You can of course still flash
+and debug using any :ref:`debug-host-tools` which work for your board
+(and which those west commands wrap).
+
+If you want to use these build system targets but do not want to
+install west on your system using ``pip``, it is possible to do so
+by manually creating a :term:`west installation`:
```

Comment:
Is it true that `west build` invokes `ninja` (among others) whereas `ninja flash` invokes`west`? I think I'm guessing why but that's still confusing.

Reply:
Yes, it is true. Thanks for pointing out that's confusing. I'll fix it.